app/oxalis/security/CombinedAuthenticatorService.scala

```diff
@@ -24,18 +26,32 @@ case class CombinedAuthenticator(actualAuthenticator: StorableAuthenticator) ext
   override def isValid: Boolean = actualAuthenticator.isValid
 }
 
+class IdentityCookieSigner extends CookieSigner {
+  override def sign(data: String): String = data
+  override def extract(message: String): Try[String] = Success(message)
```

Comment:
This means, we don't sign cookies, right? Should we?
I think we don't care if somebody changes the cookies, so this should be fine, I guess.

Reply:
I agree, this might be something to consider in the future. But we didn’t sign them before, so I guess this doesn’t block the PR